src/client/viewSandbox.js

```diff
@@ -456,7 +458,17 @@ const init = () => {
 
   // There are some timing issues between the CoralUI panelstack and CodeMirror rendering.
   // Without this, sometimes the CodeMirror editors don't render correctly.
-  document.querySelector('coral-panelstack').addEventListener('coral-panelstack:change', () => {
+  document.querySelector('.coral-TabList').addEventListener('click', (e) => {
+    if (e.target.className === 'coral-TabList') {
+      return;
+    }
```

Comment:
If we go with the `data-panel-id` strategy outlined in my other comment, we could just do `if (!e.target.hasAttribute('data-panel-id'))` here instead of relying on CSS classes (which ideally should only be used for stying purposes and not JS logic).